api/app/signals/apps/signals/models/area.py

```diff
@@ -5,6 +5,7 @@ class AreaType(models.Model):
     class Meta:
         verbose_name = 'Gebiedstype'
         verbose_name_plural = 'Gebiedstypen'
+        ordering = ['code']
```

Comment:
Forgot to run `makemigrations`?

There are now changes in the model that are not reflected in a migration. Please add the migration.

Reply:
will fix